### PR TITLE
Iss12

### PR DESCRIPTION
**Pull request by ptrrdrck**
Branched from `next12upgrade`. Contains styling updates outlined in issue #12.

### CODE REVIEW

File: src/Header.js [hunk 1]
```diff
@@ -59,58 +63,50 @@ export default function Header(props) {
...
+                  height: "38px",
+                  width: "auto",
                   borderRadius: "8px",
                 }}
-                target="_blank"
-                href="https://app.keeperdao.com/"
-                rel="noopener noreferrer"
               >
-                App
+                Enter App
```
Comment (pangolin1) [inline src/Header.js:109]:
```suggestion
                App
```
I don't think we need the word "enter" here, "blog - abut - app" is fine

**Review by pangolin1 — CHANGES_REQUESTED** [review]
Looks mostly good, great on mobile. Left one suggestion in the code and would also fix the grid layouts here:

Screen Shot 2021-11-27 at 6 23 16 PM

The top 2 paragraphs should be column direction, bold text on top of normal text (like it is on mobile). 

Below, just make sure the trade, borrow, earn divs are the same height and line up.

File: src/Header.js (hunk 1 above)
Comment (pangolin1) [reply src/Header.js:109]: on second thoughts, this is fine

**Comment by Pai-Sho** [comment]
Are we intentionally getting rid of the gradient animation on these buttons? 

https://user-images.githubusercontent.com/26280070/143785843-91904e61-1c42-4613-b9c6-b33b7e5d9f20.mov

**Comment by Pai-Sho** [comment]
Current behavior for reference

https://user-images.githubusercontent.com/26280070/143785871-5edd4a60-9fdc-4c4a-807a-aeaea64f8b07.mov

**Comment by Pai-Sho** [comment]
Styling has been removed on some buttons

Screen Shot 2021-11-28 at 2 10 40 PM

**Comment by ptrrdrck** [comment]
Yes this was intentional as that button is now self-animating its gradient. When applying the new animation I opted to mimic the state styles of the Enter App button above for continuity. This button's hover state matches the above's default, and the active states also match. I feel it looks better than jumping to a new gradient state on hover, but I'm happy to try alts. I will check on the removed styling - is that ss hiding-book.js?

**Comment by pangolin1** [comment]
This is looking good. One last major thing I'd like addressed. 

This is the landing page on my browser (16inch laptop screen):
Screen Shot 2021-11-29 at 12 18 41 PM

I don't like that the copy is cut off at the bottom of the screen. Can address this in 2 ways:
- stretch the div out to the same width as the div above it (i think this will look better anyway)
- maybe tweak the vertical spacing (it also looks a bit gappy on mobile between some of the grid elements)

You also have the option of stretching out the width of the whole page, which runs a little narrow on desktop.

**Review by pangolin1 — APPROVED** [review]

**Pull request merged**